x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -509,14 +516,26 @@ export const AnomalyExplanationDetails: FC<{ anomaly: AnomaliesTableRecord }> =
 
   return (
     <div>
-      <EuiText size="xs">
-        <h4>
-          <FormattedMessage
-            id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationTitle"
-            defaultMessage="Anomaly explanation"
-          />
-        </h4>
-      </EuiText>
+      <EuiFlexGroup gutterSize="none">
+        <EuiFlexItem>
+          <EuiText size="xs">
+            <h4>
+              <FormattedMessage
+                id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationTitle"
+                defaultMessage="Anomaly explanation"
+              />
+            </h4>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
```

Comment:
How does it look with the link directly to the right of the `Anomaly explanation` heading? On wider screens it looks a bit distant from the section it's providing info on.

<img width="811" alt="image" src="https://user-images.githubusercontent.com/7405507/215085095-c4c31597-df4e-457d-90b2-b50996bd5e3c.png">

Reply:
Moved in https://github.com/elastic/kibana/pull/149674/commits/c9be81e713c44d1a5e81dacf5fb6cff8d72afed8